gulpfile.js

```diff
@@ -149,6 +150,24 @@ function cordovaPrepare() {
   return runCommand(`cordova prepare ${platform}`);
 }
 
+function cordovaConfigureBeta() {
```

Comment:
Nit: I think this name should include a word like "maybe" or "check" to reflect its conditional action.

Reply:
Done in 50e904e.